### PR TITLE
hotfix: bring down client on id duplicated

### PR DESCRIPTION
**Pull request by moliva**
## What does this PR change?

The current PR partially addresses #4513 bringing down the client and removing disabling comms from retrying to connect to the realm when it was initially kicked due to a duplicated id error (i.e. two tabs open with the same wallet address).

## How to test the changes?

1. Launch the explorer on one tab with a given wallet
2. Launch the explorer on another tab with the same wallet
3. At least one of the two should get an error explaining the issue, bringing down the client and disconnecting from comms
4. The tab with no issue or a new tab if the previous two erred should be available to connect successfully with no realm issues

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md

### CODE REVIEW

**Review by marianogoldman — APPROVED** [review]
LGTM

**Comment by marianogoldman** [comment]
Testing it with 2 tabs... what happened is both tabs got to the same state. They both show the error, they keep playing the music (from Genesis Plaza) and they both keep high activity in the console while trying to reconnect.
Screenshot

File: browser-interface/packages/shared/loading/ReportFatalError.ts [hunk 1]
```diff
@@ -87,8 +87,6 @@ export function BringDownClientAndReportFatalError(
 ) {
   let sagaStack: string | undefined = payload['sagaStack']
 
-  debugger
```
Comment (kuruk-mm) [inline browser-interface/packages/shared/loading/ReportFatalError.ts:90]: Why are we removing this debugger? It's useful for debugging. The `debugger` only triggers if you have the console opened.

**Review by kuruk-mm — CHANGES_REQUESTED** [review]

File: browser-interface/packages/shared/loading/ReportFatalError.ts (hunk 1 above)
Comment (moliva) [reply browser-interface/packages/shared/loading/ReportFatalError.ts:90]: > Testing it with 2 tabs... what happened is both tabs got to the same state. They both show the error, they keep playing the music (from Genesis Plaza) and they both keep high activity in the console while trying to reconnect. Screenshot

Yes, I see two things with the current approach:
1. It takes some time before clients realize they need to go down
2. There's still reconnecting logic connected but that due to having a fatal error set, [should not be actually invoking the connection logic here](https://github.com/decentraland/unity-renderer/blob/hotfix/brind-down-client-on-id-duplicated/browser-interface/packages/shared/comms/sagas.ts#L413)

**Comment by moliva** [comment]
@marianogoldman 
> Testing it with 2 tabs... what happened is both tabs got to the same state. They both show the error, they keep playing the music (from Genesis Plaza) and they both keep high activity in the console while trying to reconnect. Screenshot

Yes, I see two things with the current approach:
1. It takes some time before clients realize they need to go down
2. There's still reconnecting logic connected but that due to having a fatal error set, [should not be actually invoking the connection logic here](https://github.com/decentraland/unity-renderer/blob/hotfix/brind-down-client-on-id-duplicated/browser-interface/packages/shared/comms/sagas.ts#L413)

**Pull request merged**